docs/src/howto-guides/004_connect.md

```diff
@@ -113,27 +113,32 @@ To remove existing connection and create new one follow the advice and issue [`t
 
 ```shell
 $ sudo tedge disconnect c8y
-Removing Cumulocity bridge.
+Removing C8y bridge.
```

Comment:
Why this change? `Cumulocity` sounds as the correct name and `C8y` is an acronym far less well-known than `k8s` ;-)

Reply:
Because of this commit last year. https://github.com/thin-edge/thin-edge.io/commit/81526b08d56d12d33ee98ee5c4911bc02a398eda

In `disconnect_bridge.rs`, the current code is
```
println!("Removing {:?} bridge.\n", self.cloud);
```

But if we change it to the below, it will be back to `Cumulocity`, not `c8y`. 
```
        let cloud_name: String = self.cloud.into();
        println!("Removing {} bridge.\n", cloud_name);
```

Reply:
Should I add the fix to this PR?